lookout/style/format/analyzer.py

```diff
@@ -205,7 +213,8 @@ def train(cls, ptr: ReferencePointer, config: Mapping[str, Any], data_service: D
         cls._log.info("trained %s", model)
         return model
 
-    def generate_fixes(self, data_service: DataService, changes: Sequence) -> Iterable[FixData]:
+    def generate_file_fixes(self, data_service: DataService, changes: Sequence,
```

Comment:
`changes: Sequence` of what? I see that we do not have the element type in the original code, can you please add it still?